aws_batch/pipeline.py

```diff
@@ -489,6 +500,8 @@ def run_sample(sample_s3_input_path, sample_s3_output_path,
     handler.addFilter(TimeFilter())
     handler.setFormatter(formatter)
     logger.addHandler(handler)
+    default_logparams = {"sample_s3_output_path": sample_s3_output_path,
```

Comment:
probably make this a constant and upper cading it.

Reply:
ok I uppercased it. Didn't make it global because it needs sample_s3_output_path and result_dir, which are only defined within run_sample